src/lib/components/Grid/Grid.scss

```diff
@@ -51,3 +51,14 @@
   grid-column: span var(--rui-local-column-span, 1); // 2.
   grid-row: span var(--rui-local-row-span, 1); // 2.
 }
+
```

Comment:
Just a question, but what about other elements? Documentation says: "HTML tag to render. Can be any valid HTML tag of your choice, usually a block-level element.". So wouldn't be better to reset styles of all those elements, or of all elements (`*`)?

Reply:
Honestly, I spent some time thinking about it — and I don't know. Let's go with `*` and see.